### DISCUSSION TITLE
API. Connecting to Telegram...

### DISCUSSION THREAD

**Discussion by andalbrue** [jkctech/Telegram-Trilateration#6] [General]
When I execute the file nearby.py, the script does not request phone number and Telegram password. Only show the message "Connecting to Telegram..."

**Comment by jkctech** [comment]
May I ask what operating system you are running? I had the same behaviour on Windows 7 and I could not find why this was happening. I might investigate more upcoming days.

**Reply by andalbrue** [reply]
Win10

**Reply by jkctech** [reply]
alright, will have a look at it, thanks!

**Reply by laminda** [reply]
Haven't solved this problem yet? I have exactly the same situation unfortunately

**Reply by jkctech** [reply]
Does not really matter anymore. Telegram has fixed this exploit and my software does not work anymore.

Thanks for the interest :)
